modules/pubCommonIdSystem.js

```diff
@@ -90,6 +93,33 @@ export const pubCommonIdSubmodule = {
       const callback = this.makeCallback(pixelUrl, storedId);
       return callback ? {callback: callback} : {id: storedId};
     }
+  },
+
+  /**
+   * @param {string} domain
+   * @param {HTMLDocument} document
+   * @return {(string|undefined)}
+   */
+  domainOverride: function () {
```

Comment:
Should we delete this test cookie when we know the domain as this function will be executed on each request ?

Reply:
the last successfully written cookie is removed on line 115 so it should be cleared when topDomain is returned